modules/gitops-argo-cd-properties.adoc

```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
+  * _<resources>_ - The container compute resources.
+  * _<version>_ - The tag to use with the Redis container image.
+
+|`resourceCustomizations` |Customize resource behavior.|`__<empty>__` |
+|`resourceExclusions` |Completely ignore entire classes of resource group.|`__<empty>__` |
+|`resourceInclusions` |The configuration to configure which resource group/kinds are applied.|`__<empty>__` |
+|`server` |Argo CD Server configuration options.|`__<object>__`
+a|* _<autoscale>_ - Server autoscale configuration options.
+  * _<extraCommandArgs>_ - List of arguments added to the existing arguments set by the Operator.
+  * _<grpc>_ - gRPC configuration options.
+  * _<host>_ - The hostname used for Ingress or Route resources.
+  * _<ingress>_ - Ingress configuration for the Argo CD server component.
+  * _<insecure>_ - Toggles the insecure flag for Argo CD server.
+  * _<resources>_ - The container compute resources.
+  * _<replicas>_ - The number of replicas for the Argo CD server. Must be greater than or equal to `0`. If `Autoscale` is enabled, `replicas` is ignored.
+  * _<route>_ - Route configuration options.
+  * _<service.Type>_ - The `ServiceType` used for the service resource.
+  * _<logLevel>_ - The log level to be used by the Argo CD Server component. Valid options are  `debug`, `info`, `error`, and `warn`.
+  * _<logFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
+  * _<env>_ - Environment to set for the server workloads.
+
+|`sso` |Single Sign-on options.|`__<object>__`
+a|* _<image>_ - The container image for Keycloak. This overrides the `ARGOCD_KEYCLOAK_IMAGE` environment variable.
+  * _<keycloak>_ - Configuration options for Keycloak SSO provider.
+  * _<dex>_ - Configuration options for Dex SSO provider.
+  * _<provider>_ - The name of the provider used to configure Single Sign-on. For now the supported options are Dex and Keycloak.
+  * _<resources>_ - The container compute resources.
+  * _<verifyTLS>_ - Whether to enforce strict TLS checking when communicating with Keycloak service.
+  * _<version>_ - The tag to use with the Keycloak container image.
+|`statusBadgeEnabled` |Enable application status badge.|`true` |
+
+|`tls` |TLS configuration options.|`__<object>__`
+a|* _<ca.ConfigMapName>_ - The name of the `ConfigMap` which contains the CA certificate.
+  * _<ca.SecretName>_ - The name of the secret which contains the CA Certificate and Key.
+  * _<initialCerts>_ - Initial set of certificates in the `argocd-tls-certs-cm` config map for connecting Git repositories via HTTPS.
+|`userAnonyousEnabled` |Enable anonymous user access.|`true` |
+|`version` |The tag to use with the container image for all Argo CD components.|Latest Argo CD version|
+
+|`banner` |Add a UI banner message.|`__<object>__`
+a|* _<banner.Content>_ - The banner message content (required if a banner is displayed).
+  * _<banner.URL.SecretName>_ - The banner message link URL (optional). 
```

Comment:
```suggestion
  * _<banner.url.SecretName>_ - The banner message link URL (optional). 
```
I did not see this property in the linked list, only `banner.url`: https://github.com/redhat-developer/gitops-operator/blob/master/bundle/manifests/argoproj.io_argocds.yaml#L7328 could you please check if the `secretName` part is still relevant? Thank you!